### ISSUE TITLE
Validation errors Description="Attribute 'http://schemas.openxmlformats.org/wordprocessingml/2006/main:styleId' should have unique value. Its current value 'ListTable6Colorful-Accent5' duplicates with others."

### ISSUE THREAD

**Issue by PrzemyslawKlys** [EvotecIT/OfficeIMO#85]
![image](https://user-images.githubusercontent.com/15063294/204040209-fad118b4-fd68-4c83-ba58-869ac99b47d5.png)

Basically all tables styles are reporting this error. This seems standard even for documents generated by Office 365. When you create a word document and add 1-3-50 tables to it, and add multiple styles validation errors will occur. It doesn't seem to affect anything and doesn't show up for FileFormat 2007. Only for newer formats.

**Comment by nestorjlemus** [comment]
any solution?

**Comment by PrzemyslawKlys** [comment]
No, because it doesn't really affect anything as far as I can tell and it behaves like that on any Word created by Word O365? What's not working for you?
